guides/ormi-subgraph-hosting.md

```diff
@@ -0,0 +1,71 @@
+# Using Ormi's 0xgraph Subgraph Hosting Solution 
+
+## Introduction
+
+[Ormi Labs](https://app.0xgraph.xyz/dashboard/api) offers a powerful solution for hosting and managing subgraphs, which can be particularly useful for Rollkit rollups and their ecosystems - enabling dApps with low-latency and reliable real-time data. This beginner guide aims to demonstrate the capabilities of 0xgraph's subgraph hosting solution for rollup teams. 
+
+Please note that network support for your specific rollup will need to be added before subgraphs can be fully migrated. To get network support added for your rollup, please reach out to the Ormi team at julian@ormi.xyz.
+
+This guide will walk you through the process of using Ormi's services to host your Subgraphs, providing insight into how these services can enhance the real-time data querying capabilities of your Rollkit rollups ecosystem once network support is implemented.
+
+## Prerequisites
+
+- Node.js installed on your system
+- Basic understanding of Subgraphs and GraphQL APIs
+
+## Getting Started with Ormi
+
+### 1. Install the 0xgraph CLI
+First, install the 0xgraph CLI tool:
+```bash
+npm i @xgraph/cli
+```
+First, install the 0xgraph CLI tool:
+
+### 2. Authentication
+Authenticate with Ormi using your API key:
+```bash
+0xgraph auth
+```
+When prompted, enter your deploy key. You can obtain this key by signing up at https://app.0xgraph.xyz/dashboard/api.
```

Comment:
I'd reverse this since the user is going to need the key before running the command. 
```suggestion
Next you'll need to authenticate with Ormi using your API key. You can obtain this key by signing up at https://app.0xgraph.xyz/dashboard/api.

Once you have your API key, authenticate with Ormi using the following command:

```bash
0xgraph auth
```

When prompted, enter your deploy key. 
```